ProcessMaker/Http/Controllers/Api/V1_1/CaseController.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace ProcessMaker\Http\Controllers\Api\V1_1;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use ProcessMaker\Http\Controllers\Controller;
+use ProcessMaker\Http\Requests\GetAllCasesRequest;
+use ProcessMaker\Http\Resources\V1_1\CaseResource;
+use ProcessMaker\Models\CaseStarted;
+
+class CaseController extends Controller
+{
+    protected $defaultFields = [
```

Comment:
Add documentation to clarify this are the field that are going to be selected if the main query of the endpoint.

Reply:
Documentation was added